translator/internaldata/oc_to_metrics.go

```diff
@@ -417,7 +417,7 @@ func exemplarToMetrics(ocExemplar *ocmetrics.DistributionValue_Exemplar, exempla
 	exemplar.SetValue(ocExemplar.GetValue())
 	attachments := exemplar.FilteredLabels()
 	ocAttachments := ocExemplar.GetAttachments()
-	attachments.InitEmptyWithCapacity(len(ocAttachments))
+	attachments.EnsureCapacity(len(ocAttachments))
```

Comment:
Same problem here.

Reply:
Not a problem, the output param is empty.

Reply:
How do we know that it is empty? It is not possible to tell by reading only this function code, right? Or do I miss something?

Reply:
It is not, but we do the same pattern lots of times:

func foo() {
  md := pdata.NewMetrics()
  initializeResource(md.Resource)
  ...
}

Reply:
No more problems.